response_operations_ui/views/collection_exercise.py

```diff
@@ -253,7 +260,11 @@ def _upload_collection_instrument(short_name, period):
 
```

Comment:
I wonder if it would be worth having a function before this validate method is called that determines the type and the name of collection instrument.

Then you could split the _validate_collection_instrument function in two, one for bres and one for others and call the appropriate function when you know the type of CI. I feel that might be cleaner, you could also then use the results further now and avoid to do the split multiple times.

Reply:
Would you mind if I held off on those changes until the post-BRES work?  A lot of this code is going to be moved around and reorganised and it might be a good idea to get it in there instead of now

Reply:
np :)